index.bs

```diff
@@ -4894,6 +4972,37 @@ side=], or to terminate or error the stream.
   <p>Returns a {{WritableStream}} representing the [=writable side=] of this transform stream.
 </dl>
 
+<div algorithm="TransformStream transfer steps">
+{{TransformStream}} objects are [=transferable objects=]. Their [=transfer steps=], given |value|
+and |dataHolder|, are:
+
+<!-- Find a way to linkify ReadableStream transfer steps and WritableStream transfer steps -->
+
+ 1. If ! [$IsReadableStreamLocked$](|value|.\[[readable]]) is true, throw a {{TypeError}} exception.
+ 1. If ! [$IsWritableStreamLocked$](|value|.\[[writable]]) is true, throw a {{TypeError}} exception.
+ 1. Let |readableDataHolder| be { \[[Type]]: `"ReadableStream"` }.
```

Comment:
Is the [[Type]] tagging used anywhere? I have this question generally, although in this case it seems particularly clear that it's not used.

Reply:
No. Removed.